sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+SELECT "genres"."name" AS "Genres",
+       COUNT("films") AS "# of Films"
+  FROM "actors"
+  JOIN "castMembers" USING ("actorId")
+  JOIN "films" USING ("filmId")
+  JOIN "filmGenre" USING ("filmId")
+  JOIN "genres" USING ("genreId")
+ WHERE "actors"."actorId" = 178
```

Comment:
Since you can't guarantee the actor you're looking for will have that id without performing another query (manual or otherwise), please adjust your filter to specify exactly who you're looking for:

---

![image](https://user-images.githubusercontent.com/16143771/186543783-1b6c35b3-1aa0-4d7b-bc54-70eebd868a22.png)